### ISSUE TITLE
Emails are not delivering once form is submitted. 

### ISSUE THREAD

**Issue by WIP123** [CalderaWP/Caldera-Forms#2233]
Submissions are found in entries but the emails are not delivering to individual's Outlook.

**Comment by AndyClifton** [comment]
I had a similar issue and then realized that you actually need to...

1. Capture the user's email address in the form using the "Email" field type. Set the slug to something like `e_mail`, and 
2. Under the "Email" tab for the form, set "Email Recipients" to `%e_mail%`.

Then it sends the email. This system allows you to choose which email address gets used for the reply  (imagine the use case where you want to report that an email address doesn't work, for example).

**Comment by Forev3rBlackeyed** [comment]
Hello. This is closed but I have the same problem today, November 17th 2018.
I had mails not delivering at all, or sometimes delivering and only on the main webmail not the extra recipients (only on Caldera entries).

Then I switched from Wordpress mailer to SendGrid. It all seemed to work and the emails went to the email I set and the other recipients email addresses. 

When somebody from yahoo.com sends a message through caldera though, that message doesn't get delivered neither to the main address or to the extra set addresses. Only appears on Caldera entries.
And I don't know if this happens with more addresses and if it is a bug or mis-configuration.

I would appreciate any help. 

The settings are like the ones you describe above.

From Email: %email%
Reply to Email: %email%

Antispam is on

Kind Regards,
Blackeyed
